cloudstack/VPCService.go

```diff
@@ -2855,7 +2855,7 @@ type VPC struct {
 	JobID                string               `json:"jobid"`
 	Jobstatus            int                  `json:"jobstatus"`
 	Name                 string               `json:"name"`
-	Network              []string             `json:"network"`
+	Network              []Network            `json:"network"`
```

Comment:
@tetra12 - the change LGTM, but I believe this change would also probably be required at `CreateVPCResponse` (https://github.com/apache/cloudstack-go/blob/main/cloudstack/VPCService.go#L721) and `UpdateVPCResponse` (https://github.com/apache/cloudstack-go/blob/main/cloudstack/VPCService.go#L3172). You could probably add a check in the function named `mapType` (https://github.com/apache/cloudstack-go/blob/main/generate/generate.go#L1849) - to handle this change via the code generator. Thanks.

Reply:
Hi @Pearl1594 
Thanks for your hint. I updated the generator code as per your advice. Pls check it up again